### PR TITLE
Create User, Organization and Event interfaces

### PR DESCRIPTION
**Pull request by AntonovAtanas**

### CODE REVIEW

File: src/shared/interfaces/Event.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export interface Event {
+  title: string;
+  shortDescription: string;
```
Comment (Getsov) [inline src/shared/interfaces/Event.ts:3]: to be unified with either "Descr"or "Description"

File: src/shared/interfaces/Event.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    coordinates: { lat: string; long: string };
+    city: string;
+    address: string;
+    phone: string;
+    email: string;
+  };
+  category: string;
+  likedCount: number;
+  creator: string;
+  winners: string[];
```
Comment (Getsov) [inline src/shared/interfaces/Event.ts:17]: Here isn't it better to be array of objects? So we can store names, vehicle, etc..?

File: src/shared/interfaces/Organization.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export interface Organisation {
+    name: string;
+    createdEvents: string[],
```
Comment (Getsov) [inline src/shared/interfaces/Organization.ts:3]: Here finally is it going to be an array of strings or array of objects as we've discussed? And isn't it better to be optional?

File: src/shared/interfaces/User.ts [hunk 4]
```diff
@@ -1,6 +1,10 @@
 export interface User {
-    username: string;
     email: string;
-    password: string;
+    role: string; 
+    firstName?: string; 
+    lastName?: string; 
+    city: string;
+    isDeleted: boolean;
+    likedEvents: string[];
```
Comment (Getsov) [inline src/shared/interfaces/User.ts:8]: The same question as createdEvents - Here finally is it going to be an array of strings or array of objects as we've discussed? And isn't it better to be optional?

File: src/shared/interfaces/User.ts [hunk 5]
```diff
@@ -1,6 +1,10 @@
 export interface User {
-    username: string;
     email: string;
-    password: string;
+    role: string; 
+    firstName?: string; 
+    lastName?: string; 
+    city: string;
```
Comment (Getsov) [inline src/shared/interfaces/User.ts:6]: City is better to be optional

File: src/shared/interfaces/Event.ts [hunk 6]
```diff
@@ -0,0 +1,19 @@
+export interface Event {
```
Comment (Getsov) [inline src/shared/interfaces/Event.ts:1]: id is missing and which of the properties can be optional?

File: src/shared/interfaces/Organization.ts [hunk 7]
```diff
@@ -0,0 +1,7 @@
+export interface Organisation {
```
Comment (Getsov) [inline src/shared/interfaces/Organization.ts:1]: id is missing

**Review by Getsov — CHANGES_REQUESTED** [review]

File: src/shared/interfaces/Event.ts (hunk 6 above)
Comment (AntonovAtanas) [reply src/shared/interfaces/Event.ts:1]: 1. Added id
2. Added winners and phone for contact to be optional

**Pull request closed without merging**